src/sql/base/browser/ui/table/treeGrid.ts

```diff
@@ -89,10 +89,13 @@ export class TreeGrid<T extends Slick.SlickData> extends Table<T> {
 				const rowElement = this._tableContainer.querySelector(`div [role="row"][aria-rowindex="${(i + 1)}"]`);
 				// If the row element is found in the dom, we are setting the required aria attributes for it.
 				if (rowElement) {
-					if (rowData.expanded !== undefined) {
-						rowElement.ariaExpanded = rowData.expanded;
-					} else {
-						rowElement.removeAttribute('aria-expanded');
+					const cellDiv = <HTMLElement>rowElement.querySelector(`.slick-cell.l0`);
```

Comment:
That's L and not 1. 
I am just hardcoding it to the first cell of the row. Since, everything else is hardcoded, I figured there is no harm in doing that here.

Reply:
That's just a slick grid thing, right? I would strongly suggest adding a comment explaining what this means then because it's not something I think most people are going to know without understanding the inner workings of slick grid.